api/rest/restcore/ApiEnabledMiddleware.php

```diff
@@ -0,0 +1,36 @@
+<?php
+# MantisBT - A PHP based bugtracking system
+
+# MantisBT is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# MantisBT is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A webservice interface to Mantis Bug Tracker
+ *
+ * @package MantisBT
```

Comment:
Existing SOAP files don't have a sub-package defined.  I don't think it is a big deal, but if we add it we should probably use `webservice` or `webapi` since there is a lot of shared code between SOAP and REST.